### PR TITLE
Update the README.md

### PR DESCRIPTION
**Pull request by lehoangan2906**
A small  additions to the README.md file including comments for using pip3 on macs with python versions 2 and 
Screen Shot 2022-02-25 at 10 49 39 PM
3

### CODE REVIEW

File: README.md~ [hunk 1]
```diff
@@ -0,0 +1,230 @@
+[![Downloads](http://pepy.tech/badge/opencv-python)](http://pepy.tech/project/opencv-python)
```
Comment (alalek) [inline README.md~:1]: > `README.md~`

???

---

The same unrelated file:

> `.README.md.un~`

File: README.md [hunk 2]
```diff
@@ -9,15 +9,15 @@ Check the manual build section if you wish to compile the bindings from source t
 ### Installation and Usage
 
 1. If you have previous/other manually installed (= not installed via ``pip``) version of OpenCV installed (e.g. cv2 module in the root of Python's site-packages), remove it before installation to avoid conflicts.
-2. Make sure that your `pip` version is up-to-date (19.3 is the minimum supported version): `pip install --upgrade pip`. Check version with `pip -V`. For example Linux distributions ship usually with very old `pip` versions which cause a lot of unexpected problems especially with the `manylinux` format.
+2. Make sure that your `pip` version is up-to-date (19.3 is the minimum supported version): `pip install --upgrade pip`. Check version with `pip -V`. For example Linux distributions ship usually with very old `pip` versions which cause a lot of unexpected problems especially with the `manylinux` format. (On Macos, if you have both python 2 and python 3 installed simultaneously and you want to install Opencv specifically for python3, you might want to use pip3 instead of pip).
```
Comment (alalek) [inline README.md:12]: > Macos
> Opencv
> "python 2" / "python 3"

Use correct names writing. See one line above.

**Review by alalek — COMMENTED** [review]
No real problem if Python is used properly - through virtualenv.

"virtualenv" brings `python` / `pip` commands in all cases.
It is better to learn Python ecosystem first.

**Review by PoweredTable — APPROVED** [review]

**Review by PoweredTable — APPROVED** [review]

**Comment by tanhi17** [comment]
i have this problem : 
(myenv) D:\python_opencvverif> python test.py
Traceback (most recent call last):
  File "D:\python_opencvverif\test.py", line 1, in <module>
    import cv2 as cv
  File "D:\python_opencvverif\myenv\Lib\site-packages\cv2\__init__.py", line 8, in <module>
    from .cv2 import *
ImportError: DLL load failed while importing cv2: Le module spécifié est introuvable.
